src/components/Hyperlink.tsx

```diff
@@ -17,7 +16,7 @@ export const Hyperlink: React.FC<HyperlinkProps> = ({ href, text }) => {
         navigate(href);
       }}
     >
-      {text}
+      {typeof text === "string" ? text : text()}
```

Comment:
We need to figure out how to make this work with signals, just a note no change req